pkg/envoytest/harness.go

```diff
@@ -141,12 +142,8 @@ const bootstrap = `
       {
         "name": "static_layer",
         "static_layer": {
-          "envoy.reloadable_features.enable_deprecated_v2_api": true,
-          "envoy.deprecated_features:envoy.api.v2.route.HeaderMatcher.regex_match": true,
-          "envoy.deprecated_features:envoy.api.v2.route.RouteMatch.regex": true,
-          "envoy.deprecated_features:envoy.config.filter.http.ext_authz.v2.ExtAuthz.use_alpha": true,
-          "envoy.deprecated_features:envoy.config.trace.v2.ZipkinConfig.HTTP_JSON_V1": true,
-          "envoy.reloadable_features.ext_authz_http_service_enable_case_sensitive_string_matcher": false
+					"envoy.reloadable_features.no_extension_lookup_by_name": false,
+          "re2.max_program_size.error_level": 200
```

Comment:
This is indented funny. Looks like that `envoy.reloadable_features.no_extension_lookup_by_name` line uses tabs for indent while the other lines use spaces?